app/notifications/validators.py

```diff
@@ -351,10 +358,11 @@ def validate_template(template_id, personalisation, service: Service, notificati
     check_template_is_for_notification_type(notification_type, template.template_type)
     check_template_is_active(template)
 
-    template_with_content: Template = create_content_for_notification(template, personalisation)
+    template_with_content = create_content_for_notification(template, personalisation)
```

Comment:
Does this hit the database on every notification?

Reply:
Not this line, but higher up in the `validate_template` function: https://github.com/cds-snc/notification-api/blob/main/app/notifications/validators.py#L350

Which gets the template from the db here: https://github.com/cds-snc/notification-api/blob/main/app/notifications/validators.py#L365

Reply:
Oh so we are not adding one more retrieval, as the template was already retrieved from the database prior to these changes?

Reply:
Yes that's correct!